addons/music/fnc_getMusicPlaying.sqf

```diff
@@ -1,37 +1,44 @@
 /* ----------------------------------------------------------------------------
 Function: CBA_fnc_getMusicPlaying
+
 Description:
     A function used to return the current time on playing music. Must have been started with CBA_fnc_playMusic
+    
 Parameters:
-    BOOL: if true, return bool (default: False)
+    BOOL: if true, return bool (default: false)
+
+Returns:
+    array [class, music time, time left] OR bool (true for playing music)
+    
 Example:
     (begin example)
-    _musicTime = [] call CBA_fnc_getMusicPlaying;
+        _musicTime = [] call CBA_fnc_getMusicPlaying;
     (end example)
-Returns:
-    array [class, music time, time left] OR bool (true for playing music)
+
 Author:
     Fishy
 ---------------------------------------------------------------------------- */
 #include "script_component.hpp"
 
-private ["_bool","_return","_trackTime","_remainingTime"];
-_bool = [_this,0,false,[false]] call BIS_fnc_param;
-_return = ['', 0, 0, 0];
+if (!params [["_bool",false,[false]]]) exitWith {WARNING('Incorrect param');};
+
 if (isNil QGVAR(track)) exitWith {
-    if (_bool) then {_return = false;};
-    _return
+    if (_bool) then {false} else {['', 0, 0, 0]}
 };
+
 if (_bool) exitWith {true};
 
 GVAR(track) params ["_class","_startTime","_playPos","_duration"];
+private "_trackTime";
```

Comment:
Again that weird private thingy. I won't mention it anymore for the rest of the files.